docs/getting-started-guides/kargo.md

```diff
@@ -0,0 +1,100 @@
+* [Composable](https://github.com/kubernetes-incubator/kargo/blob/master/docs/vars.md)
+  (Choice of the network plugin, for instance)
+* Support most popular Linux
+  [distributions](https://github.com/kubernetes-incubator/kargo#supported-linux-distributions)
+* Continuous integration tests
+
+To choose a tool which fits your use case the best, you may want to read this
+[comparison](https://github.com/kubernetes-incubator/kargo/blob/master/docs/comparisons.md)
+to [kubeadm](../kubeadm) and [kops](../kops).
+
+## Creating a cluster
+
+### (1/4) Ensure the underlay [requirements](https://github.com/kubernetes-incubator/kargo#requirements) are met
+
+#### Checklist
+
+* You must have cloud instances or baremetal nodes running for your future Kubernetes cluster.
+  The easiest way to achieve that is to use the [kargo-cli tool](https://github.com/kubernetes-incubator/kargo/blob/master/docs/getting-started.md).
+* Or provision baremetal hosts with a tool-of-your-choice or launch cloud instances,
+  then create an inventory file for Ansible with this [tool](https://github.com/kubernetes-incubator/kargo/blob/master/contrib/inventory_generator/inventory_generator.py).
+
+### (2/4) Compose the deployment
+
+#### Checklist
+
+* Customize your deployment by usual Ansible meanings, which is overriding default
+  data [variables](https://github.com/kubernetes-incubator/kargo/blob/master/docs/vars.md)
+  or just stick with default values (Kargo will choose Flannel networking plugin for you
+  then). This includes steps like deciding on the:
+  * DNS [configuration options](https://github.com/kubernetes-incubator/kargo/blob/master/docs/dns-stack.md)
+  * [Networking plugin](https://github.com/kubernetes-incubator/kargo#network-plugins) to use
+  * [Versions](https://github.com/kubernetes-incubator/kargo#versions-of-supported-components)
+    of components.
+  * Additional node groups like [bastion hosts](https://github.com/kubernetes-incubator/kargo/blob/master/docs/ansible.md#bastion-host) or
+    [Calico BGP route reflectors](https://github.com/kubernetes-incubator/kargo/blob/master/docs/calico.md#optional--bgp-peering-with-border-routers).
+* Plan custom deployment steps, if any, or use the default composition layer in the
+  [cluster definition file](https://github.com/kubernetes-incubator/kargo/blob/master/cluster.yml).
+  Taking the best from Ansible world, Kargo allows users to execute arbitrary steps via the
+  ``ansible-playbook`` with given inventory, playbooks, data overrides and tags, limits, batches
+  of nodes to deploy and so on.
+* For large deployments (100+ nodes), you may want to
+  [tweak things](https://github.com/kubernetes-incubator/kargo/blob/master/docs/large-deployments.md)
+  for best results.
+
+### (3/4) Run the deployment
+
+#### Checklist
+
+* Apply deployment with [kargo-cli tool](https://github.com/kubernetes-incubator/kargo/blob/master/docs/getting-started.md)
+  or ``ansible-playbook`` manual commands.
```

Comment:
I'll put those to the related Getting Started guide in Kargo, that is referenced here in a few places